ParticleObjects/particleDungeon_class.f90

```diff
@@ -49,15 +49,18 @@ module particleDungeon_class
   !!                          does not take ununiform weight of particles into account
   !!     setSize(n)        -> sizes dungeon to have n dummy particles for ease of overwriting
   !!     printToFile(name) -> prints population in ASCII format to file "name"
+  !!     printToScreen(prop,nMax,total) -> prints property to screen for up to nMax particles
+  !!     getSize()         -> returns number of particles in dungeon
   !!
   !!   Build procedures:
   !!     init(maxSize)     -> allocate space to store maximum of maxSize particles
   !!     kill()            -> return to uninitialised state
   !!
   type, public :: particleDungeon
     private
-    real(defReal),public :: k_eff = ONE ! k-eff for fission site generation rate normalisation
-    integer(shortInt)    :: pop = 0     ! Current population size of the dungeon
+    real(defReal),public     :: k_eff = ONE   ! k-eff for fission site generation rate normalisation
+    integer(shortInt)        :: pop = 0       ! Current population size of the dungeon
+    real(defreal),public     :: endOfStepTime ! Time at end of current time step - only used in IMC calculations
```

Comment:
I don't think this is strictly necessary here. In principle that can remain entirely in the physics package, surely?